examples/go/fib_test.go

```diff
@@ -8,10 +8,12 @@ func BenchmarkFib10(b *testing.B) {
 	for i := 0; i < b.N; i++ {
 		var _ = Fib(10)
 	}
+	b.ReportMetric(3.0, "auxMetricUnits")
 }
 
 func BenchmarkFib20(b *testing.B) {
 	for i := 0; i < b.N; i++ {
 		var _ = Fib(20)
 	}
+	b.ReportMetric(4.0, "auxMetricUnits")
 }
```

Comment:
I would actually suggest creating a third benchmark function that reports additional metrics instead of modifying the existing ones
```suggestion
	for i := 0; i < b.N; i++ {
		var _ = Fib(10)
	}
}

func BenchmarkFib20(b *testing.B) {
	for i := 0; i < b.N; i++ {
		var _ = Fib(20)
	}
}

func BenchmarkFib20WithAuxMetric(b *testing.B) {
	for i := 0; i < b.N; i++ {
		var _ = Fib(20)
	}
	b.ReportMetric(4.0, "auxMetricUnits")
}
```